### ISSUE TITLE
De novo gene expression generation

### ISSUE THREAD

**Issue by cecileherbermann** [bowang-lab/scGPT#192]
Hi, 

Congratulations on the Nature paper. I was wondering if it is only possible to generate gene expression data - after pre-training and finetuning - when the cell embedding is given as input, or if you can also generate new gene expression data by using gene tokens as input. 

Kind regards,
Cecile

**Comment by subercui** [comment]
Hi, thank you for the question! For the first question, yes, it is possible. The model is currently trained in similar ways, in the sense that the loss function is to reconstruct gene expression. For the second question, I am a bit confused about what information would be used to specify the cell condition if there is no gene expression available.

**Comment by cecileherbermann** [comment]
Hi, thank you for your reply! I'm looking to fine-tune scGPT on a dataset with cells sequenced at different time points, with the goal of predicting gene expression on days that were not sequenced. For example, I want to predict gene expression for day 7 using data from day 0 and day 14. Although we don't have cell embeddings for day 7, the model should be able to infer this based on the other days. How would you recommend proceeding with this using scGPT?

**Comment by Silaschuwen** [comment]
Hi! I'm encountering the same issue recently. Sorry to bother you, but did you happen to find a solution?
